code/lib/instrumenter/src/instrumenter.ts

```diff
@@ -576,7 +582,7 @@ export class Instrumenter {
       }
 
       const hasPrevious = logItems.some((item) =>
-        [CallStates.DONE, CallStates.ERROR].includes(item.status)
+        ([CallStates.DONE, CallStates.ERROR] as (CallStates | undefined)[]).includes(item.status)
```

Comment:
It's not clear why you are type-casting here: what was the type-error otherwise?

Reply:
The reason is that `item.status` might be undefined. And the `includes` type is infamously strict. 
I agree, I don't like `as` as well @ryami333.
We could just do a simple equality check instead:
` item.status === CallStates.DONE || item.status === CallStates.ERROR`